biodatasets/psytar/psytar.py

```diff
@@ -0,0 +1,483 @@
+with the drugs were reported.
+
+This dataset can be used for:
+
+1. (multi-label) sentence classification, across 5 labels:
+    Adverse Drug Reaction (ADR)
+    Withdrawal Symptoms (WDs)
+    Sign/Symptoms/Illness (SSIs)
+    Drug Indications (DIs)
+    Drug Effectiveness (EF)
+    Drug Infectiveness (INF)
+    and Others (not applicable)
+
+2. Recognition of 5 different types of entity:
+    ADRs (4813 mentions)
+    WDs (590 mentions)
+    SSIs (1219 mentions)
+    DIs (792 mentions)
+
+In the source schema, systematic annotation with UMLS and SNOMED-CT concepts are provided.
+"""
+
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from utils import parsing, schemas
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{Zolnoori2019,
+  author    = {Maryam Zolnoori and
+               Kin Wah Fung and
+               Timothy B. Patrick and
+               Paul Fontelo and
+               Hadi Kharrazi and
+               Anthony Faiola and
+               Yi Shuan Shirley Wu and
+               Christina E. Eldredge and
+               Jake Luo and
+               Mike Conway and
+               Jiaxi Zhu and
+               Soo Kyung Park and
+               Kelly Xu and
+               Hamideh Moayyed and
+               Somaieh Goudarzvand},
+  title     = {A systematic approach for developing a corpus of patient \
+               reported adverse drug events: A case study for {SSRI} and {SNRI} medications},
+  journal   = {Journal of Biomedical Informatics},
+  volume    = {90},
+  year      = {2019},
+  url       = {https://doi.org/10.1016/j.jbi.2018.12.005},
+  doi       = {10.1016/j.jbi.2018.12.005},
+}
+"""
+
+_DATASETNAME = "psytar"
+
+_DESCRIPTION = """\
+The "Psychiatric Treatment Adverse Reactions" (PsyTAR) dataset contains 891 drugs
+reviews posted by patients on "askapatient.com", about the effectiveness and adverse
+drug events associated with Zoloft, Lexapro, Cymbalta, and Effexor XR.
+
+This dataset can be used for (multi-label) sentence classification of Adverse Drug
+Reaction (ADR), Withdrawal Symptoms (WDs), Sign/Symptoms/Illness (SSIs), Drug
+Indications (DIs), Drug Effectiveness (EF), Drug Infectiveness (INF) and Others, as well
+as for recognition of 5 different types of named entity (in the categories ADRs, WDs,
+SSIs and DIs)
+"""
+
+_HOMEPAGE = "https://www.askapatient.com/research/pharmacovigilance/corpus-ades-psychiatric-medications.asp"
+
+_LICENSE = "CC BY 4.0"
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.TEXT_CLASSIFICATION]
+
+_SOURCE_VERSION = "1.0.0"
+_BIGBIO_VERSION = "1.0.0"
+
+
+@dataclass
+class PsyTARBigBioConfig(datasets.BuilderConfig):
+    schema: str = "source"
+    name: str = "psytar_source"
+    version: datasets.Version = _SOURCE_VERSION
+    description: str = "PsyTAR source schema"
+    subset_id: str = "psytar"
```

Comment:
my understanding is that you still want to `from utils.configs import BigBioConfig` and use it in `BUILDER_CONFIGS` below